modules/task/execution.go

```diff
@@ -37,14 +37,14 @@ type fnRegister struct {
 
 var fnLookup = fnRegister{fnNameMap: make(map[string]interface{})}
 
-// fnSignature represents a function and its arguments
-type fnSignature struct {
+// FnSignature represents a function and its arguments
+type FnSignature struct {
```

Comment:
Backend implementations and tests need access to it and they can be in different repos and different modules.